src/js/store/appState/types.ts

```diff
@@ -1,13 +1,42 @@
+import Point from 'esri/geometry/Point';
+
 export interface LeftPanel {
   tabViewVisible: boolean;
   activeTab: string;
   openLayerGroup: string;
 }
 
+interface SpecificAreaResults {
+  area?: number;
```

Comment:
see if these can be required and if nothing breaks

Reply:
@vaidotasp @SwampGuzzler just a quick update - making these properties required causes TS errors where the logic assigns them as empty objects. Unfortunately union typing doesn't resolve those errors, so I'll keep them optional for now